HW_14/test_human.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from HW_14.human import Human
+
+
+def test_name_private(create_human):
+    """Test name is private"""
+    human = create_human
+    with pytest.raises(AttributeError, match=f"'{human.__class__.__name__}' object has no attribute '__name'"):
+        assert 'Sam' == human.__name
+
+
+def test_status_private(create_human):
+    """Test status is private"""
+    human = create_human
+    with pytest.raises(AttributeError, match=f"'{human.__class__.__name__}' object has no attribute '__status'"):
+        assert 'alive' == human.__status
+
+
+def test_age_limit_private(create_human):
+    """Test age_limit is private"""
+    human = create_human
+    with pytest.raises(AttributeError, match=f"'{human.__class__.__name__}' object has no attribute '__age_limit'"):
+        assert '100' == human.__age_limit
+
+
+def test_age_property(create_human):
+    """Test age property"""
+    human = create_human
+    assert 18 == human.age
```

Comment:
why 18? where do you get this magic number?

Reply:
18 that is the age value used in fixture "create_human". 
That is why it was used as a specific value for assert. 
Should it be better created as separate variable inside the test, in order to make it more explicit?